src/common/RootProviders.tsx

```diff
@@ -24,7 +25,7 @@ export function RootProviders({
   const { apiBaseURL, theme } = useDeveloperConfig();
 
   return (
-    <PersistedQueryProvider>
+    <QueryClientProvider client={new QueryClient()}>
```

Comment:
We'll want to declare this query client in the top-level global scope -- it needs to be stable for the life of the app, and as-is will re-create on each render.

Reply:
Gah, totally - I had this sneaking feeling that I was missing something.